dbms/src/Flash/Coprocessor/tests/gtest_ti_remote_block_inputstream.cpp

```diff
@@ -276,7 +276,7 @@ struct MockReceiverContext
 
     void makeAsyncReader(
         const Request &,
-        std::shared_ptr<AsyncReader> &,
+        std::unique_ptr<AsyncReader> &,
         grpc::CompletionQueue *,
         UnaryCallback<bool> *) const {}
```

Comment:
maybe
```
std::unique_ptr<AsyncReader> makeAsyncReader(
        const Request &,
        std::shared_ptr<AsyncReader> &,
        grpc::CompletionQueue *,
        UnaryCallback<bool> *) const {}
```
is better?

Reply:
> maybe
> 
> ```
> std::unique_ptr<AsyncReader> makeAsyncReader(
>         const Request &,
>         std::shared_ptr<AsyncReader> &,
>         grpc::CompletionQueue *,
>         UnaryCallback<bool> *) const {}
> ```
> 
> is better?

I think this may cause compile error as it's function signature is different from the GRPCReceiverContext's.

Reply:
ok